java/ql/src/semmle/code/java/frameworks/spring/SpringStringUtils.qll

```diff
@@ -0,0 +1,58 @@
+import java
+private import semmle.code.java.dataflow.ExternalFlow
+
+private class SpringStringUtilsModel extends SummaryModelCsv {
+  override predicate row(string row) {
+    row = [
+        "org.springframework.util;StringUtils;false;addStringToArray;;;Argument[0];ReturnValue;taint",
+        "org.springframework.util;StringUtils;false;addStringToArray;;;Argument[1];ReturnValue;taint",
+        "org.springframework.util;StringUtils;false;addStringToArray;;;Argument[1];Argument[0];taint",
```

Comment:
`addStringToArray` copies the input, so this one doesn't hold

Reply:
Well that just seems like poor practice :joy: